config/7.8.0/pim/process-migration.yaml

```diff
@@ -0,0 +1,132 @@
+## KIE ProcessMigration BEGIN
+processMigration:
+  deploymentConfigs:
+    - metadata:
+        name: "[[.ApplicationName]]-process-migration"
+        labels:
+          app: "[[$.ApplicationName]]"
+          application: "[[$.ApplicationName]]"
+          service: "[[.ApplicationName]]-process-migration"
+      spec:
+        replicas: 1
+        selector:
+          deploymentConfig: "[[.ApplicationName]]-process-migration"
+        strategy:
+          type: Recreate
+        triggers:
+          - type: ImageChange
+            imageChangeParams:
+              automatic: true
+              containerNames:
+                - "[[.ApplicationName]]-process-migration"
+              from:
+                kind: ImageStreamTag
+                namespace: openshift
+                name: "[[.ProcessMigration.Image]]:[[.ProcessMigration.ImageTag]]"
```

Comment:
image triggers with imagestreams are the old way of deploying. since we've moved to image sha's and disconnected installs, we default to direct image url pulls instead using -
https://github.com/kiegroup/kie-cloud-operator/blob/b675341b04e1d0cac9553f001be32e528731413d/config/7.8.0/pim/process-migration.yaml#L40

reference other object yaml like console and smart router to determine how this is done.
https://github.com/kiegroup/kie-cloud-operator/blob/b675341b04e1d0cac9553f001be32e528731413d/pkg/controller/kieapp/defaults/defaults.go#L261-L279

you'll probably want to add an ImageURL field to PIM template object.
https://github.com/kiegroup/kie-cloud-operator/blob/b675341b04e1d0cac9553f001be32e528731413d/config/7.8.0/common.yaml#L13-L41

Reply:
Updated to use the new way of deploying